website/docs/getting-started/setup/installation-guides/kubernetes/install-k8s-business-edition.mdx

```diff
@@ -0,0 +1,29 @@
+# Kubernetes
```

Comment:
```suggestion
# Deploy on Kubernetes
```

Reply:
@pranayagarwal96 - Same as above. Please confirm.

Reply:
The headers are not needed as confirmed by @pranayagarwal96